packages/sui-pde/src/adapters/optimizely/multiple.js

```diff
@@ -0,0 +1,92 @@
+import OptimizelyAdapter from './index.js'
+
+let defaultAdapterId
+
+class MultipleOptimizelyAdapter {
+  static createMultipleOptimizelyInstances(optionsByInstance) {
+    return Object.keys(optionsByInstance).reduce((acc, adapterId) => {
+      let {datafile, sdkKey, ...restOptions} = optionsByInstance[adapterId]
+
+      if (
+        !datafile &&
+        typeof window !== 'undefined' &&
+        window.__INITIAL_CONTEXT_VALUE__?.pde[adapterId]
+      ) {
+        datafile = window.__INITIAL_CONTEXT_VALUE__.pde[adapterId]
+        sdkKey = undefined
+      }
+
+      acc[adapterId] = OptimizelyAdapter.createOptimizelyInstance({
+        datafile,
+        sdkKey,
+        ...restOptions
+      })
+
+      return acc
+    }, {})
+  }
+
+  #adapters = {}
+
+  constructor(optimizelyAdapters) {
+    defaultAdapterId = Object.keys(optimizelyAdapters)[0] // first adapter will be the defaultAdapter
```

Comment:
If `optimizelyAdapters` object keys are adapterId's, why not find for `default` one first?

Reply:
adapterIds are not required to have a `default` key, you could pass:
```
const multiple = new MultipleOptimizelyAdapter({
   adapter1: firstAdapter,
   adapter2: secondAdapter,
})
```

In this case, the `defaultAdapterId` would be `adapter1` so when using PDE without passing any `adapterId` in the calls, `adapter1` will be used.